assets/javascripts/discourse/templates/components/chat-message.hbs

```diff
@@ -114,6 +114,14 @@
 
               {{format-chat-date message details}}
 
+              {{#if bookmark}}
```

Comment:
I imagine we have the same kind of logic in core, this could be a {{bookmark-icon bookmark=bookmark}} component, which would also be easy to test

Reply:
Actually in core all of the icons and this kind of logic for bookmarks are set in widgets. However this is a good idea, I will come back to it after merging the polymorphic bookmarks in core.